src/components/Capture/capture.js

```diff
@@ -23,7 +24,8 @@ class Capture extends Component {
       hasWebcam: DetectRTC.hasWebcam,
       DetectRTCLoading: true,
       uploadFallback: false,
-      fileError: false
+      fileError: false,
```

Comment:
by looking at the code, it might be simpler to just have one property called `error`, which has both `fileError` and `serverError`.